<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XQueryDialog</class>
 <widget class="QDialog" name="XQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluate X-PATH</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Your Query</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Current X-PATH :</string>
        </property>
        <property name="buddy">
         <cstring>m_xpathTextEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_xpathTextEdit"/>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_pushButtonCount">
          <property name="text">
           <string notr="true">count()</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButtonSum">
          <property name="text">
           <string notr="true">sum()</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButtonFormatNumber">
          <property name="text">
           <string notr="true">format-number()</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButtonConcat">
          <property name="text">
           <string notr="true">concat()</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButtonNormalize">
          <property name="text">
           <string notr="true">normalize-space()</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButtonNot">
          <property name="text">
           <string notr="true">not()</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Query :</string>
        </property>
        <property name="buddy">
         <cstring>m_queryTextEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="XinxCodeEditAction" name="m_queryTextEdit" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>&amp;Result (encapsulated in the Result balise)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="XinxCodeEditAction" name="m_resultTextEdit" native="true">
        <property name="readOnly" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_benchLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XinxCodeEditAction</class>
   <extends>QWidget</extends>
   <header location="global">editors/xinxcodeedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushButtonCount</sender>
   <signal>clicked()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>addFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushButtonSum</sender>
   <signal>clicked()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>addFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushButtonFormatNumber</sender>
   <signal>clicked()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>addFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushButtonConcat</sender>
   <signal>clicked()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>addFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushButtonNormalize</sender>
   <signal>clicked()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>addFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushButtonNot</sender>
   <signal>clicked()</signal>
   <receiver>XQueryDialog</receiver>
   <slot>addFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFunction()</slot>
 </slots>
</ui>
